phpunit.xml.dist

```diff
@@ -9,6 +9,7 @@
 >
     <php>
         <ini name="error_reporting" value="-1" />
+        <server name="APP_SECRET" value="FOO" />
```

Comment:
If the need arises to have an `env.test` (db tests most likely), this will no longer be hard coded..

Reply:
removed - caching issue

done